os/arch/arm/src/artik05x/src/artik05x_tash.c

```diff
@@ -76,7 +76,11 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-
+#ifndef CONFIG_SMARTFS_MULTI_ROOT_DIRS
```

Comment:
How about below?
```
#ifdef CONFIG_XXX_DIRS
#define XXX YYY"d1"
#else
#define XXX YYY
#endif
```

Because positive is better to understand quickly and prevent misunderstanding easily than negative.

Reply:
okay